slides/payment_systems.tex

```diff
@@ -0,0 +1,195 @@
+% Choose one to switch betweeen slides and handout
+%\documentclass[]{beamer}
+\documentclass[handout]{beamer}
+
+% Video Meta Data
+\title{Bitcoin, Blockchain and Cryptoassets}
+\subtitle{Introduction: Payment Systems}
+\author{Prof. Dr. Fabian Schär}
+\institute{University of Basel}
+
+% Config File
+\input{../config/config.tex}
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+\begin{document}
+
+\thispagestyle{empty}
+\begin{frame}[noframenumbering]
+	\titlepage
+\end{frame}
+
+%%%
+\begin{frame}{Payment Systems: Today's Challenge}
+
+Physical Cash: Presence in the same place at the same time
+\begin{figure}[h]
+	\center
+		\input{../assets/figures/cash_payment.tex}
+\end{figure}
+
+\vspace{1.5 em}
+
+\uncover<1->{Digital Cash: Proof of ownership
+\begin{figure}[h]
+	\center
+		\input{../assets/figures/ecash_payment.tex}
+\end{figure}}
+	
+\end{frame}
+%%%	
+
+%%%
+\begin{frame}{Today's Solution: Intermediaries Keeping Money Registries}
+
+\begin{figure}[h]
+	\center
+		\input{../assets/figures/registry_payment.tex}
+\end{figure}
+
+\uncover<1->{\vspace{1 em}
+
+Registry Types\\
+\vspace{1 em}
+\textbf{Implicit:} Verbal agreements, limited to tight-knit groups.\\
+\textbf{Explicit:} Records in written or digital databases, no limits.}
+	
+\end{frame}
+%%%	
+
+%%%
+\begin{frame}{Implicit Registry Example: The Stone Money of Yap}
+
+\begin{figure}[h]
+	\center
+		\input{../assets/figures/yap_system.tex}
+\end{figure}
+	
+\end{frame}
+%%%
+
+%%%
+\begin{frame}{Closed Loop Payment Systems}
+
+\textbf{Defining Characteristics}
+ 	\begin{itemize}
+ 		\item Direct relationships with both, payer and payee.
+ 		\item One operator, one set of rules.
+ 	\end{itemize}
+\vspace{1.5em}
+
+\uncover<1->{\textbf{Examples of Closed Loop Systems}
+ 	\begin{itemize}
+ 		\item Merchant issued payment cards
+ 		\item Western Union
+ 		\item PayPal
+ 		\item Several mobile payment services: MPesa, WeChat		
+ 	\end{itemize}}
+\vspace{1.5em}
+
+\uncover<2->{
+$\Rightarrow$ Highly efficient but utility depends on number of end-users.
+}
+	
+\end{frame}
+%%%	
+
+%%%
+\begin{frame}{Open Loop Payment Systems}
+
+\begin{figure}[h]
+	\center
+		\input{../assets/figures/ol_payment_system.tex}
+\end{figure}
+\vspace{1em}
+
+\uncover<1->{\textbf{Multiple relationships and rule sets lead to compounding of:}
+ 	\begin{itemize}
+ 		\item Processing time (from switch to settlement)
+ 		\item Fees (subscription and transaction based)
+ 		\item Network dependencies
+ 		\item Exception handling cost (fraud, errors, underfunding)		
+ 	\end{itemize}}
+
+	
+\end{frame}
+%%%	
+
+%%%
+\begin{frame}{Network Examples}
+
+\textbf{SWIFT}, global scope, since 1970s\\
+Standardized transaction \textit{messaging }system.
+
+\vspace{1em}
+
+\textbf{SEPA}, EU and EFTA, since 2008/2009\\
+Payment network for low cost EUR payments at minimum speed.
```

Comment:
What do you mean with minimum speed?

Reply:
There is a rule that a SEPA payment must be credited to the payee's account within max. 3 business days.

Reply:
Ok. Maybe it would be clearer to refer to it as "maximum transmission time" or something along those lines.

Reply:
Included as minor change in commit 36a01b9